tools/pd-ctl/pdctl/command/global.go

```diff
@@ -57,6 +57,8 @@ func InitHTTPSClient(caPath, certPath, keyPath string) error {
 	return nil
 }
 
+type convertOutputFunc func(string) string
```

Comment:
I think there is no need to define `convertOutputFunc`.

Reply:
I see. We can use `:=` for declaration